notes/health-care-evaluation.vbs

```diff
@@ -2975,13 +2978,610 @@ DO
 			Call validate_MAXIS_case_number(err_msg, "*")
 			If HC_form_name = "Select One..." Then err_msg = err_msg & vbCr & "* Select the form received that you are processing a Health Care evaluation from."
 			If IsDate(form_date) = False Then err_msg = err_msg & vbCr & "* Enter the date the form being processed was received."
+			'Add validation to make sure LTC field is blank for ex parte
+            If HC_form_name = "No Form - Ex Parte Determination" AND ltc_waiver_request_yn <> "N/A - Ex Parte Process" THEN err_msg = err_msg & vbCr & "* Select 'N/A - Ex Parte Process' for the LTC/Waiver Services field."
 			If trim(worker_signature) = "" Then err_msg = err_msg & vbCr & "* Enter your name to sign your CASE/NOTE."
 			IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine
 		End If
 	LOOP UNTIL err_msg = ""
 	CALL check_for_password_without_transmit(are_we_passworded_out)
 Loop until are_we_passworded_out = false
 
+'Add ex parte SQL connections and script
+If HC_form_name = "No Form - Ex Parte Determination" Then
```

Comment:
All of the functionality needs to be nested in this if statement so that this and only this run when Ex Parte is selected.

Reply:
Makes sense - I made that change